pvlib/tests/test_modelchain.py

```diff
@@ -1455,6 +1455,23 @@ def test_aoi_model_no_loss(sapm_dc_snl_ac_system, location, weather):
     assert mc.results.ac[1] < 1
 
 
+def test_aoi_model_interp(sapm_dc_snl_ac_system, location, weather, mocker):
+    # similar to test_aoi_models but requires arguments to work, so we
+    # add 'interp' aoi losses model arguments to module
+    sapm_dc_snl_ac_system.arrays[0].module_parameters['iam_ref'] = (1., 0.85)
+    sapm_dc_snl_ac_system.arrays[0].module_parameters['theta_ref'] = (0., 80.)
+    mc = ModelChain(sapm_dc_snl_ac_system, location,
+                    dc_model='sapm', aoi_model='interp',
+                    spectral_model='no_loss')
+    m = mocker.spy(sapm_dc_snl_ac_system, 'get_iam')
+    mc.run_model(weather=weather)
+    assert m.call_count == 1
```

Comment:
`assert_called_once_with()` expects all arguments provided, which includes `aoi` and is computed with `weather` data, so the new test checks that the kwargs passed to `iam.interp` are `iam_ref` and `theta_ref` only. Wanted to bring that up since that the call to interp does not require to be done with kwargs but PVSystem does so.